eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/AuthorConverter.php

```diff
@@ -75,7 +76,7 @@ public function toFieldDefinition(StorageFieldDefinition $storageDef, FieldDefin
     {
         $fieldDef->fieldTypeConstraints->fieldSettings = new FieldSettings(
             [
-                'defaultAuthor' => $storageDef->dataInt1,
+                'defaultAuthor' => $storageDef->dataInt1 ? $storageDef->dataInt1 : AuthorType::DEFAULT_VALUE_EMPTY,
```

Comment:
Hmm, ok, that's unexpected (my initial investigation showed this to be 0), but indeed FieldType must be prepared for it.

How about using PHP7 null coalescing operator?
```php
'defaultAuthor' => $storageDef->dataInt1 ?? AuthorType::DEFAULT_VALUE_EMPTY,
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2432/commits/24ba6ea85a4c65bc22d1b8fb79feae36a5631337.